src/modules/commander/state_machine_helper.cpp

```diff
@@ -119,7 +119,6 @@ const char *const nav_state_names[vehicle_status_s::NAVIGATION_STATE_MAX] = {
 	"6: unallocated",
 	"7: unallocated",
 	"AUTO_LANDENGFAIL",
-	"AUTO_LANDGPSFAIL",
```

Comment:
@sfuhrer the entries in this nav_state_names array need to stay consistent with the nav_state constants in vehicle_status.msg

Reply:
thanks for raising: https://github.com/PX4/PX4-Autopilot/pull/18586